api/src/iosMain/kotlin/com.giancarlobuenaflor.kflogger/backend/system/KSimpleLoggerBackend.darwin.kt

```diff
@@ -0,0 +1,130 @@
+package com.giancarlobuenaflor.kflogger.backend.system
+
+import com.giancarlobuenaflor.kflogger.KLogger
+import com.giancarlobuenaflor.kflogger.backend.KLogData
+import com.giancarlobuenaflor.kflogger.parser.KPrintfMessageParser
+import com.giancarlobuenaflor.kflogger.toOsLogType
+import kotlinx.cinterop.ptr
+import platform.darwin.__dso_handle
+import platform.darwin._os_log_internal
+import platform.darwin.os_log_create
+import platform.darwin.os_log_t
+import kotlin.native.concurrent.AtomicReference
+
+/** A logging backend that uses the `OSLog` class to output log statements. */
+public actual class KSimpleLoggerBackend actual constructor(public val logger: KLogger) : KAbstractBackend(logger) {
+
+  public actual override fun log(data: KLogData) {
+    val templateContext = data.getTemplateContext()
+    data.getTemplateContext()?.getParser()
```

Comment:
What does this do? It seems like this is left over from previous iterations of your code